packages/trader/src/Modules/Trading/Components/Form/TradeParams/Accumulator/accumulators-info-display.jsx

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { localize } from '@deriv/translations';
+import Fieldset from 'App/Components/Form/fieldset.jsx';
+import { connect } from 'Stores/connect';
+import { Money, Text } from '@deriv/components';
+import classNames from 'classnames';
+
+const AccumulatorsInfoDisplay = ({ currency, maximum_payout, maximum_ticks }) => {
+    const labels = [localize('Maximum payout'), localize('Maximum ticks')].map((label, index) => (
+        <Text key={index} size='xxs' weight='bold'>
```

Comment:
Can we use this `key` here and by this way for `values` and component below?
```suggestion
        <Text key={label} size='xxs' weight='bold'>
```

Reply:
@NikitK-deriv IMHO there is no harm in using the array’s index as “key” in “static” lists - those whose items number and order stay the same like we have here. I agree with the author's conclusion: https://www.developerway.com/posts/react-key-attribute
Do you agree? 🙂 @matin-binary @ali-hosseini-fs @yashim-deriv @eduard-deriv